npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -234,6 +236,8 @@ def path_dependency_details_from_manifest(file)
           select { |_, v| v.is_a?(String) && v.start_with?(*path_starts) }.
           map do |name, path|
             path = path.gsub(PATH_DEPENDENCY_CLEAN_REGEX, "")
+            raise PathDependenciesNotReachable, name if path.start_with?("/")
```

Comment:
Same as above. This would help _me_, so I'm suggesting it 😇 If you think it's noise, or if it's going to log information that shouldn't be, feel free to 🗑️ it.

```suggestion
            raise PathDependenciesNotReachable, "#{name} at #{path}" if path.start_with?("/")
```